test/useObservableProps.test.tsx

```diff
@@ -10,15 +10,20 @@ function toJson(val: any): string {
     return JSON.stringify(val)
 }
 
-const { rerender } = render(<div />)
-
 function doTest(options: UseObservablePropsMode<any>) {
     describe(`options: ${JSON.stringify(options)}`, async () => {
         const shallow = options === "shallow"
 
         let computedCalls: string[] = []
         let renders!: number
 
+        let rerender: (ui: React.ReactElement<any>) => void
+        beforeAll(() => {
+            cleanup()
```

Comment:
Nice attempt, but that won't do :D Running cleanup in `beforeAll` is totally useless.

Reply:
well, it is a function so it will run between each of the two "doTest" calls at least :)